packages/tools/src/commands/oss/index.js

```diff
@@ -0,0 +1,221 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+/* eslint-disable no-await-in-loop */
+
+const { execFile } = require(`child_process`);
+const { promisify } = require(`util`);
+const { Command } = require(`clipanion`);
+const fs = require('fs-extra');
+const inquirer = require('inquirer');
+const path = require('path');
+const glob = require('glob');
+const chalk = require('chalk');
+
+const templates = require('./templates');
+
+const execFileP = promisify(execFile);
+const maxBuffer = 1024 * 1024;
+
+const execute = (cmd, args, cwd) => execFileP(cmd, args, { maxBuffer, cwd, encoding: 'utf-8' });
+
+const NAME = 'build-plugin';
+const ROOT = path.join(__dirname, '../../../../../');
```

Comment:
You should be able to use `process.env.PROJECT_CWD` with yarn (no need to change this path every time the file moves around)

Reply:
🤯